src/fondant/compiler.py

```diff
@@ -173,10 +173,14 @@ def _generate_spec(
             if extra_volumes:
                 volumes.extend(extra_volumes)
 
+            ports: t.List[t.Union[str, dict]] = []
+            ports.append("8787:8787")
```

Comment:
Perhaps we can use a more dynamic approach. For now, I have just used the default ports of the dashboard.

Reply:
if we go for the client approach and don't revert back to the standard approach then we can always just expose the default port. Any reason why you might pick a different or multiple ports?

Reply:
I have separated the code and created a meaningful variable. This makes it clearer what the port represents.